src/util/storagePool.tsx

```diff
@@ -59,3 +72,15 @@ export const testDuplicateStoragePoolName = (
     },
   ];
 };
+
+export const isPowerflexIncomplete = (
+  formik: FormikProps<StoragePoolFormValues>,
+): boolean => {
+  return (
+    formik.values.driver === powerFlex &&
+    (!formik.values.powerflex_domain ||
+      !formik.values.powerflex_pool ||
+      !formik.values.powerflex_gateway ||
+      !formik.values.powerflex_user_password)
```

Comment:
isn't `powerflex.user.name` also required based on the spec?

Reply:
Lxd has a default notion of the user being "admin". So it is optional as you only need to set it if it deviates from that default.